### ISSUE TITLE
Panic with "Failed to parse patch version"

### ISSUE THREAD

**Issue by hasali19** [ClementTsang/bottom#574]
Running btm panics with:

```
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Other("Failed to parse patch version")', /cargo/registry/src/github.com-1ecc6299db9ec823/procfs-0.9.1/src/lib.rs:303:34
   0: bottom::panic_hook
   1: std::panicking::rust_panic_with_hook
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:595:17
   2: std::panicking::begin_panic_handler::{{closure}}
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:497:13
   3: std::sys_common::backtrace::__rust_end_short_backtrace
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/sys_common/backtrace.rs:141:18
   4: rust_begin_unwind
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:493:5
   5: core::panicking::panic_fmt
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/core/src/panicking.rs:92:14
   6: core::result::unwrap_failed
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/core/src/result.rs:1355:5
   7: std::sync::once::Once::call_once::{{closure}}
   8: std::sync::once::Once::call_inner
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/sync/once.rs:418:21
   9: procfs::process::stat::Stat::from_reader
  10: <core::iter::adapters::filter_map::FilterMap<I,F> as core::iter::traits::iterator::Iterator>::next
  11: futures_executor::local_pool::run_executor::{{closure}}
  12: futures_executor::local_pool::block_on
  13: std::sys_common::backtrace::__rust_begin_short_backtrace
  14: core::ops::function::FnOnce::call_once{{vtable.shim}}
  15: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/alloc/src/boxed.rs:1546:9
      <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/alloc/src/boxed.rs:1546:9
      std::sys::unix::thread::Thread::new::thread_start
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/sys/unix/thread.rs:71:17
  16: start_thread
  17: <unknown>
```

I did a bit of investigating and it seems that this is caused by procfs panicking if the kernel patch version is > 255 (4.9.277-117 on my system). I believe this should be fixed on the latest version by https://github.com/eminence/procfs/pull/137, so updating that should hopefully fix this.

**Comment by ClementTsang** [comment]
Thanks for finding and looking into this!  I'll look into deploying a patch shortly.

**Comment by ClementTsang** [comment]
Change pushed to master branch, will do a release in a bit.

**Comment by ClementTsang** [comment]
Released in 0.6.4.  Thanks for catching this!
